tutorials/imad-tutorial-pt1/index.ipynb

```diff
@@ -0,0 +1,2078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.                                   .filter(ee.Filter.contains(rightValue=aoi,leftField='.geo')) 
</pre><p> </p><p>Are both region filters necessary (i.e. <code>filterBounds</code> and <code>ee.Filter.contains</code>)? My guess is you can just use <code>ee.Filter.contains</code>. This applies to <code>im1</code> and <code>im2</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Yes, but it takes an orders of magnitude longer to find regions and then times out when trying to plot the them. Left unchanged.

Reply:
Interesting. Acknowledged no change.